configure.ac

```diff
@@ -499,7 +499,6 @@ then
     test "$enable_aesofb" = "" && enable_aesofb=yes
     test "$enable_aescfb" = "" && enable_aescfb=yes
     test "$enable_aescbc_length_checks" = "" && enable_aescbc_length_checks=yes
-    test "$enable_camellia" = "" && enable_camellia=yes
```

Comment:
Please revert the changes to `snip Camellia, IDEA, and HC128 out of the all sets`. The removal of these is being handled by individual PR's and will avoid future conflicts.

Reply:
isn't it OK to just snip them out of the `all` sets but leave otherwise as-is?  if I don't do that I'll need to do the whole temporary-suppressions runaround on them for the `clang-tidy` `--enable-all` tests, since they're not fixed up to make `clang-tidy` happy.

p.s. I forgot to include Rabbit in the omission list.  so either I'll turn the other 3 back on, or turn Rabbit off too, your call.

Reply:
Might want to do a rebase against master.

Reply:
nm, I just remembered I refactored `--enable-all` scenarios in `wolfssl-multi-test` to use `"${ENABLE_ALL_TEST_FLAGS[@]}"` so I'll just add `--disable-hc128` etc to that and revert the configure.ac tweak.

Anthony's #4767 would otherwise conflict.

Reply:
@anhu indeed -- I _did_ rebase, and picked up the HC128 excision (woohoo!).

Reply:
:)